wsdl/ver10/schema/onvif.xsd

```diff
@@ -1807,12 +1807,106 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>Volume setting of the output. The applicable range is defined via the option AudioOutputOptions.OutputLevelRange.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="EQPresetToken" type="tt:ReferenceToken" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Optional audio EQ preset token to be applied to the audio output.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
 					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
 				</xs:sequence>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+
+	<xs:simpleType name="FrequencyRange">
+    <xs:restriction base="xs:string">
+        <xs:enumeration value="20Hz-60Hz">
+            <xs:annotation>
+                <xs:documentation>Sub-bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="60Hz-250Hz">
+            <xs:annotation>
+                <xs:documentation>Bass frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="250Hz-500Hz">
+            <xs:annotation>
+                <xs:documentation>Low midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="500Hz-2kHz">
+            <xs:annotation>
+                <xs:documentation>Midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="2kHz-4kHz">
+            <xs:annotation>
+                <xs:documentation>Upper midrange frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+        <xs:enumeration value="4kHz-20kHz">
+            <xs:annotation>
+                <xs:documentation>High frequencies.</xs:documentation>
+            </xs:annotation>
+        </xs:enumeration>
+    </xs:restriction>
+</xs:simpleType>
+
+	<!--===============================-->
+	<!--          FrequencyGainPair            -->
+	<!--===============================-->
+	<xs:complexType name="FrequencyGainPair">
+    	<xs:sequence>
+        <xs:element name="FrequencyRange" type="xs:string">
+            <xs:annotation>
+                <xs:documentation>Defined in tt:FrequencyRange, e.g., 20Hz-60Hz, 2kHz-4kHz.(Readonly)</xs:documentation>
+            </xs:annotation>
+        </xs:element>
+        <xs:element name="Gain" type="xs:float">
+            <xs:annotation>
+                <xs:documentation>Normalized value (0.0 ~ 1.0).</xs:documentation>
```

Comment:
Question on this scale, what do this normalized value mean? Is it linear, logarithmic? Only attenuation, not amplifiation?

Reply:
Updated this to decibel based on Bangkok f2f discussion. Please re-check the updated commit.